lib/src/architectureIntegrationTest/kotlin/com/lemonappdev/konsist/architecture3/Architecture3Test.kt

```diff
@@ -0,0 +1,106 @@
+package com.lemonappdev.konsist.architecture3
+
+import com.lemonappdev.konsist.api.Konsist
+import com.lemonappdev.konsist.core.architecture.Layer
+import com.lemonappdev.konsist.core.exception.KoCheckFailedException
+import com.lemonappdev.konsist.core.exception.KoPreconditionFailedException
+import com.lemonappdev.konsist.core.filesystem.PathProvider
+import com.lemonappdev.konsist.core.verify.assert
+import org.amshove.kluent.shouldThrow
+import org.amshove.kluent.withMessage
+import org.junit.jupiter.api.Test
+
+class Architecture3Test {
+    private val rootPath = PathProvider.getInstance().rootProjectPath
+
```

Comment:
nie wiem czy ten `architecure3` ma teraz sens, gdy dodałam nowy package `circulardependency`. Ewentualnie można zostawić te dwa ostatnie testy?

Reply:
Remove all of this